content/docs/command-reference/import-url.md

```diff
@@ -337,3 +351,58 @@ $ dvc repro
 Running stage 'prepare' with command:
 	python src/prepare.py data/data.xml
 ```
+
+## Example: Import straight to the remote
+
+When you have a massive dataset in a distant location, and working on a computer
+which can't actually store it locally (due to not having enough disk space) but
+you still want to take it under control of DVC just like in the scenario of
+importing it and then pushing it to the remote, then you can use `--to-remote`
+flag.
+
+It will try to import the data into the remote storage that you choose, and when
+you or any of your colleagues want to copy the data to their systems, they could
+just simply [pull](/doc/command-reference/remote). Let's do a simple example
+
+We initalize 2 directories, one being the remote storage unit and the other one
+is the workspace.
+
+```dvc
+$ mkdir /tmp/dvc-import-url-straight-to-remote/
+$ mkdir /tmp/remote
+$ cd /tmp/dvc-import-url-straight-to-remote/
+$ git init
+$ dvc init
+$ dvc remote add tmp_remote /tmp/remote
+```
+
+For transferring a source from a remote location, to the given remote you can
+combine `import-url` with `--to-remote` option which basically does the whole
+importing and [push](/doc/command-reference/push)ing operation under the hood
```

Comment:
Again, very accurate, but it seems like too much text, probably. We can just quickly explain the next step. Something like:

> Now let's create an import .dvc file without downloading the target data, but transferring directly to remote storage instead:

And that's it, onto the next code block ⬇️

Reply:
After the code block where `dvc import-url` is used, THEN we can explain what we did 🙂 (explain the option and the argument sent to it).